spec/daru/io/importers/redis_spec.rb

```diff
@@ -1,44 +1,44 @@
-# @note
-#
-#   Custom matchers +belong_to+ and +contain_from+ have been used,
-#   as Redis doesn't necessarily insert keys in the given order. Due
-#   to this, some rows and columns might be jumbled, and there is no
-#   way to expect for an exact match while testing on RSpec. Rather,
-#   the DataFrame is tested to have the same data, in *ANY* order.
-#
-#   Signed off by @athityakumar on 08/16/2017 at 10:00PM IST
 RSpec::Matchers.define :belong_to do |expected|
-  match { |actual| (actual.to_a.uniq - expected.to_a.uniq).empty? }
+  match { |actual| (expected.to_a.uniq - actual.to_a.uniq).empty? }
 end
 
-RSpec::Matchers.define :unordered_dataframe do |expected|
+RSpec::Matchers.define :unordered_data do |expected|
   match do |actual|
     actual = actual.to_a.map { |x| x.data.to_a }.flatten.uniq
     expected.map!(&:values) unless expected.first.is_a? Array
     expected = expected.flatten.uniq
-    (actual - expected).empty?
+    (expected - actual).empty?
   end
 end
 
-RSpec.shared_examples 'redis importer' do
-  it_behaves_like 'daru dataframe'
-  its(:data)    { is_expected.to unordered_dataframe(expected_data) }
-  its(:ncols)   { is_expected.to eq(ncols)                          }
-  its(:nrows)   { is_expected.to eq(nrows)                          }
-  its(:index)   { is_expected.to belong_to(expected_index)          }
-  its(:vectors) { is_expected.to belong_to(expected_vectors)        }
+RSpec.shared_examples 'unordered daru dataframe' do |data: nil, nrows: nil, ncols: nil, order: nil, index: nil, name: nil, **opts| # rubocop:disable Metrics/LineLength
+  it_behaves_like 'a daru dataframe',
+    name: name,
+    nrows: nrows,
+    ncols: ncols,
+    **opts
+
+  its(:data)    { is_expected.to unordered_data(data)        } if data
```

Comment:
"expected to unordered data" reads strange. Maybe "match unordered" or something like this?..